lxc/info.go

```diff
@@ -141,6 +141,17 @@ func (c *infoCmd) containerInfo(d *lxd.Client, name string, showLog bool) error
 			fmt.Printf(diskInfo)
 		}
 
+		// CPU usage
+		cpuInfo := ""
+		if cs.CPU.Usage != 0 {
+			cpuInfo += fmt.Sprintf("    %s: %v\n", i18n.G("CPU usage (in seconds)"), cs.CPU.Usage / 1000000000)
```

Comment:
You'll need to run "make i18n" and commit the result, otherwise the testsuite will fail :)